feature-pack/src/main/resources/modules/system/layers/base/org/wildfly/extension/microprofile/opentracing-smallrye/main/module.xml

```diff
@@ -39,6 +40,8 @@
         <module name="org.wildfly.security.elytron"/>
 
         <module name="javax.enterprise.api" />
+        <module name="io.opentracing.opentracing-api"/>
         <module name="org.wildfly.microprofile.opentracing-smallrye"/>
+        <module name="io.jaegertracing.jaeger" optional="true"/>
```

Comment:
Please add a comment noting what makes it required, e.g.

<!-- Only needed if a jaeger-tracer is configured -->

Reply:
Well it will be required as it is required by adding the legacy behaviour now.